<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectWindow</class>
 <widget class="QDialog" name="ProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>254</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>254</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string notr="true">Project</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="archLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>A&amp;rchitecture:</string>
     </property>
     <property name="buddy">
      <cstring>archiComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="archiComboBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>x86 - 32 bit</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>x86_64 - 64 bit</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Pro&amp;ject name:</string>
     </property>
     <property name="buddy">
      <cstring>nameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="nameEdit">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter the project's name (min 3 symbols)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QFileEdit" name="pathEdit">
     <property name="toolTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter the project's folder location</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="mode">
      <enum>QFileEdit::OpenFolderDialog</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="assemblyLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Assembly options:</string>
     </property>
     <property name="buddy">
      <cstring>assemblyOptionsEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="assemblyOptionsEdit">
     <property name="placeholderText">
      <string>Additional NASM options</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="linkingLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Lin&amp;king options:</string>
     </property>
     <property name="buddy">
      <cstring>linkingOptionsEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="linkingOptionsEdit">
     <property name="placeholderText">
      <string>Additional GCC/Clang options</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="verboseBuild">
     <property name="text">
      <string>Verbose build</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="pathLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Project location:</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>nameLabel</zorder>
  <zorder>nameEdit</zorder>
  <zorder>assemblyLabel</zorder>
  <zorder>assemblyOptionsEdit</zorder>
  <zorder>linkingLabel</zorder>
  <zorder>linkingOptionsEdit</zorder>
  <zorder>verboseBuild</zorder>
  <zorder>buttonBox</zorder>
  <zorder>archLabel</zorder>
  <zorder>pathEdit</zorder>
  <zorder>pathLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFileEdit</class>
   <extends>QLineEdit</extends>
   <header>qfileedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>onShowOpenFile()</slot>
  <slot>onProjectTextChanged(QString)</slot>
 </slots>
</ui>
